app/src/main/res/layout/activity_landing_page.xml

```diff
@@ -12,6 +13,10 @@
         android:text="@string/title_landing_page"
         android:textSize="18sp"/>
 
+    <android.support.v4.widget.Space
+        android:layout_width="wrap_content"
+        android:layout_height="20dp"/>
```

Comment:
Maybe this value can be stored in dimens.xml.

A good practice is to save all margin, padding, etc... as constant in dimens.xml.

Same comment for text size.

```
<resources>
    <dimen name="action_button_min_width">56dp</dimen>
    <dimen name="indeterminate_progress_size">32dp</dimen>
    <dimen name="txt_small">16sp</dimen>
    <dimen name="txt_normal">18sp</dimen>
    <dimen name="txt_large">20sp</dimen>
</resources>
```